projects/packages/waf/src/class-rest-controller.php

```diff
@@ -108,18 +108,25 @@ public static function update_waf( $request ) {
 
 		// IP Lists Enabled
 		if ( isset( $request[ Waf_Rules_Manager::IP_LISTS_ENABLED_OPTION_NAME ] ) ) {
+			_deprecated_argument( __METHOD__, '$next-version$', 'Use jetpack_waf_allow_list_enabled and jetpack_waf_block_list_enabled instead.' );
```

Comment:
To me, using `_deprecated_argument` here feels like "best practice" so I've included it in the initial draft for this PR. However, there are a lot of WordPress sites out there running with `WP_DEBUG` enabled in production, and I experienced this during my last support rotation.

It may be better to "silently" deprecate these features using a comment, as it is still possible for older versions of Jetpack or Protect to use the latest package version when one plugin is updated and the other is not.

Reply:
I've gone ahead and removed this.